rollup.config.mjs

```diff
@@ -44,6 +46,14 @@ export default [
           { src: 'package.json', dest: 'dist' },
         ],
       }),
+      del({ targets: 'dist/*' }),
```

Comment:
suggestion: move this up first in the plugins array, even it doesn't need to, just for the sake of readability

Reply:
Done: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/7/commits/f332ea1976b8b8928986085970af107313ded53e